interface/ceed-qfunction.c

```diff
@@ -220,6 +228,22 @@ int CeedQFunctionGetFOCCA(CeedQFunction qf, char* *focca) {
   return 0;
 }
 
+/**
+  @brief Get the User Function for a CeedQFunction
+
+  @param qf              CeedQFunction
+  @param[out] f          Variable to store user function
+
+  @return An error code: 0 - success, otherwise - failure
+
+  @ref Advanced
+**/
+
+int CeedQFunctionGetUserFunction(CeedQFunction qf, int* (*f)) {
```

Comment:
I'm not super happy with how this works, but it works.

Reply:
This `f` is not a pointer to a function, just a pointer to data. You could use `void (**f)(void)` if you want a pointer to a generic function pointer.

Reply:
I will give that a go. I spent quite a while fighting this particular few lines.